requirements.txt

```diff
@@ -12,11 +12,12 @@ ujson
 # Testing
 autoflake
 black
-codecov
+databases[sqlite]
 isort
 mypy
 pytest
 pytest-cov
+sqlalchemy
```

Comment:
This is a requirement of `databases` anyways - we don't need to include it additionally, right?

Reply:
ok

Reply:
fixed.